jobs/RunCwr/config.xml

```diff
@@ -58,7 +58,7 @@ function cleanup_model() {
 }
 trap cleanup_model EXIT
 
-cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
+env MATRIX_OUTPUT_DIR=/srv/artifacts cwr -F $CONTROLLER:test-cwr-$CHARM_NAME-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
```

Comment:
@petevg i agree with keeping stuff simple regarding params, but i still think `MATRIX_OUTPUT_DIR` belongs closer to the thing calling matrix -- that is, cwr / bundletester.

I don't see how doing this would be "dropping CI specifics" into cwr/bt/matrix.  We just so happen to equate the jenkins build number with the cwr test-id, but I would argue that you'd want matrix results in your cwr/bt output whether you're running cwr by itself or as part of this ci system.

Anywho, after a run through of the cwr datastore code, it does seem like your way is the easiest way to get what we want.  I will hold onto hope that long term, we'll rework cwr to setup a result dir prior to the tests, and use that dir in a bt arg that is also honored by matrix.